DesignSafe-CI/portal
# task/DES-2255: Update apps logos

## Overview: ##
Update app icons for DS apps via an updated font file.

## PR Status: ##


## Related Jira tickets: ##

* [DES-2255](https://jira.tacc.utexas.edu/browse/DES-2255)

## Summary of Changes: ##
- Update mapping of existing icons from new font file
- Add new icon mappings
- Remove old, unused `WorkSpaceIcons.ttf` file

## Testing Steps: ##
1. Go to https://designsafe.dev/rw/workspace/
2. Click around on the tabs and ensure the new icons are visible
    - dakota
    - ansys
    - clawpack
    - swbatch
    - stko

## UI Photos:
<img width="1294" alt="Screen Shot 2022-04-06 at 10 25 58 AM" src="https://user-images.githubusercontent.com/20326896/162010896-1ad1a30b-555f-453d-b8f8-bbfa52755042.png">
<img width="1238" alt="Screen Shot 2022-04-06 at 10 23 26 AM" src="https://user-images.githubusercontent.com/20326896/162010904-2ba95755-c502-4bc3-9e08-28cc4b3ed379.png">
